### ISSUE TITLE
About the results in your paper

### ISSUE THREAD

**Issue by wyz-gitt** [Jingkang50/OpenPSG#63]
There are 2186 images of test in the  psg.json,in the log file 2177 images in the testing and verification phase, but 5000 images of val2017 in the coco file. In your paper, are the results obtained from 5000 test/verification photos or 2177 test/verification photos?
（I mean, how many pictures of training set and test set/verification set are used to get the results in your paper?the  psg.json only marks 2186 test images）

**Comment by Jingkang50** [comment]
The paper reports the result from 2177 test pictures. 
In fact, there are actually 2186 images in test set, but 9 images does not have relations, so actually they are not used.
Due to the competition, you do not have access to the entire test set. We will release it in the mid-Oct when the preliminary round passes.
